rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -258,6 +258,16 @@ public void logReleaseBtcRequestRejected(String sender, Coin amount, RejectedPeg
         this.logs.add(new LogInfo(BRIDGE_CONTRACT_ADDRESS, encodedTopics, encodedData));
     }
 
+    @Override
+    public void logBatchPegoutCreated(BtcTransaction btcTx, byte[] rskTxHash) {
```

Comment:
```suggestion
    public void logBatchPegoutCreated(BtcTransaction btcTx, List<Keccak256> rskTxHashes) {
```
Perhaps it's a better idea to receive the list of hashes and serialize them here. What do you think?

We should serialize each hash and concatenate them into a single byte[] object

Reply:
Done